src/EditorFeatures/CSharpTest/SimplifyTypeNames/SimplifyTypeNamesTests_FixAllTests.cs

```diff
@@ -79,10 +79,10 @@ static System.Int32 F(System.Int32 x, System.Int16 y)
 
 class Program
 {
-    static int F(int x, System.Int16 y)
+    static int F(int x, short y)
     {
         int i1 = 0;
-        System.Int16 s1 = 0;
+        short s1 = 0;
```

Comment:
IMO, this behavior is strictly better.  The user the "prefer intrinsic types" option set.  They then ask to "fix all" the suggestion to use predefined types in tihs file.  But we would only fix up one type of type.  I can't imagine this being more desirable than the user actually getting all cases fixed up where they can use a predefined type given that that's their optoin and they said 'fix all'.